### DISCUSSION TITLE
How to get visibility status (Public/Private) in search results?

### DISCUSSION THREAD

**Discussion by mozilla0** [microsoft-search/pnp-modern-search#2804] [Q&A]
I am trying to integrate the visibility status of groups and teams (Public/private) into the search query so that I can both display it and use it for filtering. The information is written to a standard propertyBag (GroupType) on the SiteCollection, but is not available afterwards to create a crawled property for it.

![Crawled_Properties](https://user-images.githubusercontent.com/17600172/221148374-5aede856-fe85-4f8e-bfb1-fdad6d03368d.png)
![PropertyBag_GroupType](https://user-images.githubusercontent.com/17600172/221148088-4c41d304-88da-4003-9d46-307db11c9787.png)
![SearchQueryTool_Results](https://user-images.githubusercontent.com/17600172/221148094-ec08a4db-c5db-42c9-a6d9-6ec3b8715638.png)

**Comment by wobba** [comment]
Have you tried https://pnp.github.io/powershell/cmdlets/Add-PnPIndexedProperty.html?
